src/Compiler/Microsoft.AspNetCore.Razor.Language/src/DefaultRazorIntermediateNodeLoweringPhase.cs

```diff
@@ -125,6 +130,21 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument)
         }
 
         codeDocument.SetDocumentIntermediateNode(document);
+
+        static bool TryRemoveGlobalPrefixFromDefaultUsing(in UsingReference usingReference, out UsingReference trimmed)
```

Comment:
📝 It's difficult to review the impact of this change in isolation because baselines weren't updated with each change

Reply:
That's because this change didn't cause baseline updates, it failed even before that - there was a new warning about duplicate usings.